cypress/e2e/smoke/pending_actions.cy.js

```diff
@@ -0,0 +1,68 @@
+const SAFE = 'gor:0xCD4FddB8FfA90012DFE11eD4bf258861204FeEAE'
+
+describe('Pending actions', () => {
+  before(() => {
+    cy.connectE2EWallet()
+
+    cy.visit('/welcome')
+
+    // Close cookie banner
+    cy.contains('button', 'Accept all').click()
+
+    // Ensure wallet is connected to correct chain via header
+    cy.contains('E2E Wallet @ Görli')
+  })
+
+  beforeEach(() => {
+    cy.restoreLocalStorageCache()
+  })
+
+  afterEach(() => {
+    cy.saveLocalStorageCache()
+  })
+
+  it('should add the Safe with the pending actions', () => {
+    // Enters Loading Safe form
+    cy.contains('button', 'Add existing Safe').click()
+    cy.contains('Connect wallet & select network')
+
+    // Inputs the Safe address
+    cy.get('input[name="address"]').type(SAFE)
+    cy.contains('Next').click()
+
+    cy.contains('Owners and confirmations')
+    cy.contains('Next').click()
+
+    cy.contains('Add').click()
+  })
+
+  it('should display the pending actions in the Safe list sidebar', () => {
+    cy.get('aside').within(() => {
+      cy.get('[data-testid=ChevronRightIcon]').click({ force: true })
+    })
+
+    cy.get('li').within(() => {
+      cy.contains('0xCD4F...eEAE').should('exist')
+
+      cy.get('img[alt="E2E Wallet logo"]').next().contains('3').should('exist')
+      cy.get('[data-testid=CheckIcon]').next().contains('3').should('exist')
+
+      // click on the pending actions
+      cy.get('[data-testid=CheckIcon]').next().click()
+    })
+  })
+
+  it('should have the right number of queued and signable transactions', () => {
+    // Navigates to the tx queue
+    cy.contains('h3', 'Transactions').should('be.visible')
+
+    // contains 3 queued transactions
+    cy.get('span:contains("out of 2")').should('have.length', 3)
```

Comment:
I think we should bee explicit here.

```suggestion
    cy.get('span:contains("1 out of 2")').should('have.length', 3)
```

Reply:
I left that part out of the assertion because it brings complexity to assert the queue size. For example if one transaction has `1 out of 2` and the next one has `2 out of 2`.

Reply:
All [queued transactions in that Safe](https://safe-web-core.staging.5afe.dev/transactions/queue?safe=gor:0xCD4FddB8FfA90012DFE11eD4bf258861204FeEAE) only have one signature each though.